src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +67,62 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            {
+                if (regNsbRoot == null)
+                {
+                    //Check for the presence of HKCU:SOFTWARE\ParticularSoftware
+                    using (var regParticularRoot = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+                    {
+                        if (regParticularRoot == null)
+                        {
+                            return true;
+                        }
+                    }
+                }
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Adding the async void to explicitly have a fire and forget model for invoking the web api.
+        /// </summary>
+        async void TrackFirstTimeUsageEvent()
```

Comment:
I disagree. Async void is for fire & forget asynchronous IO bound operations. `Task.Run` would be an abuse because you are combining an explicitly acquired worker thread with an IO bound operation.